src/client/spoke-context.tsx

```diff
@@ -1,7 +1,7 @@
-import React, { useContext, useState } from "react";
+import React, { useContext, useMemo, useState } from "react";
 
 import {
-  TexterOrganizationSettingsFragmentFragment,
+  OrganizationSettingsFragmentFragment,
```

Comment:
This should be `OrganizationSettingsFragment`

Reply:
Ah, I did not realize. I haven't seen this anywhere else, so thought it was something overlooked 🙈